tool/debian/singa-cuda8.0-cudnn5/DEBIAN/control

```diff
@@ -0,0 +1,14 @@
+Package: singa-cuda8
+Version: 1.0.1
```

Comment:
I see. Then please update the top cmake file to change the version ID to 1.1.0.
thanks.